src/pages/QuestionPage.tsx

```diff
@@ -126,52 +185,64 @@ function QuestionPage() {
-                value={answer}
-                onChange={(e) => setAnswer(e.target.value)}
-              />
-              <Button type="submit">Cek Jawaban 🧐</Button>
-            </form>
-            <Separator className=" mb-4" />
-            <div className=" grid grid-cols-1 sm:grid-cols-2 gap-4 ">
-              {answers.map((a, i) => (
-                <Answer
-                  key={i}
-                  answer={a.answer}
-                  score={a.score}
-                  isCorrect={a.isCorrect!}
-                  onClick={() => {
-                    setAnswers((prev) =>
-                      prev.map((ans, index) => {
-                        if (index === i) {
-                          return {
-                            ...ans,
-                            isCorrect: true
-                          }
-                        }
-                        return ans
-                      })
-                    )
-                  }}
+          <div>
+              {loading ? (
+                <Skeleton className="h-8 w-full sm:w-[1000px] mb-4" />
+              ) : (
+                <>
+                  <div className=" p-6 w-full sm:w-[800px] mb-4">
+                    <h2 className="text-3xl mb-4 text-center text-gray-700">
+                     {questionText}
+                    </h2>
+                  </div>
+                </>
+              )}
+            </div>
+            <div className=" mb-16 mx-auto w-full">
+              <form
+                className=" flex flex-col sm:flex-row gap-2 mb-4"
+                onSubmit={checkAnswer}
+              >
+                <Input
+                  type="text"
+                  placeholder="Apa Jawaban Kamu?"
+                  value={answer}
+                  onChange={(e) => setAnswer(e.target.value)}
                 />
-              ))}
+                <Button type="submit">Cek Jawaban 🧐</Button>
+              </form>
+              <Separator className=" mb-4" />
+              <div className=" grid grid-cols-1 sm:grid-cols-2 gap-4 ">
+                {answers.map((a, i) => (
+                  <Answer
+                    key={i}
+                    answer={a.answer}
+                    score={a.score}
+                    isCorrect={a.isCorrect!}
+                    onClick={() => {
+                      setAnswers((prev) =>
+                        prev.map((ans, index) => {
+                          if (index === i) {
+                            return {
+                              ...ans,
+                              isCorrect: true
+                            }
+                          }
+                          return ans
+                        })
+                      )
+                    }}
+                  />
+                ))}
+              </div>
+            </div>
+            <div className="flex items-center gap-4">
+              <PreviousButton />
+              {questionId !== 0 && (
```

Comment:
use shadcn separator instead

Reply:
help me to change this abang, bacislly i am backend engineer :(

Reply:
follow this doc https://ui.shadcn.com/docs/components/separator